### PR TITLE
Dashboard 1.1

### PR DESCRIPTION
**Pull request by WillWeston94**
____ Wrote Tests ____ Implemented ____ Reviewed

Necessary checkmarks:

    [√] All Tests are Passing

    [√] The code will run locally

Type of change

    [√] New feature
    [] Bug Fix

Implements/Fixes:

    description closes #

Check the correct boxes

    [√] This broke nothing
    [] This broke some stuff
    [] This broke everything

Testing Changes

    [] No Tests have been changed
    [] Some Tests have been changed
    [√] All of the Tests have been changed(Please describe what in the world happened)

Checklist:

    [] My code has no unused/commented out code
    [√] I have reviewed my code
    [] I have commented my code, particularly in hard-to-understand areas
    [√] I have fully tested my code

Notes:

User Show Page Initial Dashboard Test: Added a test case to ensure that the user show page's initial dashboard loads correctly and passes.

Rails Helper for New Episodes: Created a Rails helper method to record new episodes for VCR, enhancing our testing capabilities.

Attribute Name Correction: Replaced @movie_details.image with poster_path for consistency and clarity.

Movie_db_service Update: Replaced instances of connection with conn in the Movie_db_service for improved readability.

Movie PORO Attribute Reader: Added poster_path to the list of attributes in the Movie PORO, enhancing data access.

Application Record Strict Load: Enabled strict loading for Application Record to enforce data integrity.

Movie Facade Method Update: Replaced instance methods with class methods for the movie_facade to improve efficiency.

User Show Action Enhancement: Included viewing parties in the User show action to provide a more comprehensive user experience.

Tests for MovieDbService: Added tests for the MovieDbService to ensure it functions correctly.

PORO Tests: Created tests for various POROs (Review, Movie, Cast) and ensured they all pass.

Viewing Party Model Test: Created tests for the ViewingParty model and ensured they pass.

User Model Test: Created tests for the User model and ensured they pass.

Movie Model Test: Created tests for the Movie model and ensured they pass.

Serialize Genres in Movie Model (Pending): Added serialization for genres in the Movie model to improve data handling.

Keys for MovieFacade: Added keys to the MovieFacade for better data access.

User Show Action Update: Included User includes for the show action on UsersController to optimize database queries.

Controller Method Name Update: Renamed details to movie_details in the MoviesController show method for consistency.

Invited Friends Method: Added the invited_friends method to the ApplicationController for inheritance.

Database Migration and Schema Update: Made necessary changes to the database schema to accommodate poster_paths in objects.

User Show Dash Update: Updated the User Show Dashboard to incorporate the changes and ensure it's passing.

Rating in Review PORO: Added a rating attribute to the Review PORO.

Guests Method in ViewingParty: Added a guests method to the ViewingParty model for better data access.

User Model Associations: Added associations to the User model to establish relationships with other entities.

### CODE REVIEW

**Comment by WillWeston94** [comment]
Only 1 conflict and it was on the expected show action of MoviesController. It is now cleared

File: app/controllers/application_controller.rb
Comment (williamfchen) [inline app/controllers/application_controller.rb:1]: wow wouldn't have thought to put that here, nice!

File: app/models/movie.rb
Comment (williamfchen) [inline app/models/movie.rb:1]: I'll have to look up this one

File: app/views/users/show.html.erb
Comment (williamfchen) [inline app/views/users/show.html.erb:1]: super cool!

**Review by williamfchen — APPROVED** [review]
Nice work!!!

**Pull request merged**